website/docs/docs/cloud-integrations/downstream-exposures-tableau.md

```diff
@@ -0,0 +1,107 @@
+- Automatic &mdash; Pulled automatically for supported dbt Cloud integrations. dbt Cloud automatically creates and visualizes downstream exposures, removing the need for manual YAML definitions. These downstream exposures are stored in dbt’s metadata system, appear in [dbt Explorer](/docs/collaborate/explore-projects), and behave like manual exposures, however they don’t exist in YAML files.
+
+By leveraging downstream [exposures](/docs/build/exposures) automatically, data teams can:
+
+- Gain a better understanding of how models are used in downstream analytics, improving governance and decision-making.
+- Reduce incidents and optimize workflows by linking upstream models to downstream dependencies.
+- Automate exposure tracking for supported BI tools, ensuring lineage is always up to date.
+- [Orchestrate exposures](/docs/cloud-integrations/orchestrate-exposures) to refresh the underlying data sources during scheduled dbt jobs, improving timeliness and reducing costs. Orchestrating exposures is essentially a way to ensure that your BI tools are updated regularly by using the [dbt Cloud job scheduler](/docs/deploy/job-scheduler). See the [previous page](/docs/cloud-integrations/downstream-exposures) for more info.
+
+:::info Tableau Server
+If you're using Tableau Server, you need to [allowlist dbt Cloud's IP addresses](/docs/cloud/about-cloud/access-regions-ip-addresses) for your dbt Cloud region.
+:::
+
+## Prerequisites
+
+To configure downstream exposures automatically, you should meet the following:
+
+1. Your environment and jobs are on a supported [release track](/docs/dbt-versions/cloud-release-tracks) dbt.
+2. You have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).
+3. You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 
+4. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings.
+5. Use Tableau as your BI tool and enable metadata permissions or work with an admin to do so. Compatible with Tableau Cloud or Tableau Server with the Metadata API enabled.
+
+### Considerations
+import ConsiderationsTableau from '/snippets/_auto-exposures-considerations-tb.md';
+
+<ConsiderationsTableau/>
+
+## Set up downstream exposures
+
+Set up downstream exposures in [Tableau](#set-up-in-tableau) and [dbt Cloud](#set-up-in-dbt-cloud) to ensure that your BI tool's extracts are updated automatically.
+
+### Set up in Tableau
+
+This section of the document explains the steps you need to set up downstream exposures integration with Tableau. Once you've set this up in Tableau and [dbt Cloud](#set-up-in-dbt-cloud), you can [view the downstream exposures](#view-downstream-exposures) in dbt Explorer.
```

Comment:
It might be beneficial to lead with any admin requirements
```suggestion
This section explains the steps to configure the integration in Tableau. Once configured in both Tableau and [dbt Cloud](#set-up-in-dbt-cloud), you can [view the downstream exposures](#view-downstream-exposures) in dbt Explorer. These steps will require a Tableau site admin to complete. 
```

Reply:
good shout! [double checking that here](https://dbt-labs.slack.com/archives/C04N364M8QY/p1742205410421569)